### PR TITLE
Rdm 6555

### PR DESCRIPTION
**Pull request by MSancaktutar**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDM-6555

### Change description ###
Address Current known security vulnerabilities in ccd-case-ui-toolkit

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,6 +38,7 @@
     "test:codacy-upload": "echo `cat ./coverage/lcov.info | codacy-coverage`",
     "test:coverage": "istanbul check-coverage --statements 80 --lines 80 --functions 80 --branches 80",
     "test:watch": "karma start --auto-watch --no-single-run",
+    "test:audit": "./bin/run-yarn-audit.sh -f yarn-audit-known-issues",
```
Comment (smathangi) [inline package.json]: you've to call this from Travis file

Comment (andrewfolga) [reply package.json]: I second that. Needs to be in travis file too.

Comment (MSancaktutar) [reply package.json]: Sure. I just missed that.

File: bin/run-yarn-audit.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set +e
+yarn audit
+result=$?
+set -e
+
+if [[ "$result" != 0 ]]; then
+  if [[ -f yarn-audit-known-issues ]]; then
+    set +e
```
Comment (smathangi) [inline bin/run-yarn-audit.sh]: you might have to add blank file `yarn-audit-known-issues`

**Comment by andrewfolga** [comment]
Please make sure RDM-6556 branch of man web once importing this does not fail to load and is operational locally aka quick smoke test for basic screens to check no error in console etc.

File: bin/run-yarn-audit.sh (hunk 2 above)
Comment (MSancaktutar) [reply bin/run-yarn-audit.sh]: Added. Thank you.

**Pull request merged**